Sources/VirtualTerminal/Core/TextMetrics.swift

```diff
@@ -79,6 +81,8 @@ extension UnicodeScalar {
       return isWideCharacter ? 2 : 1
     }
     return CharType & C3_FULLWIDTH == C3_FULLWIDTH ? 2 : 1
+#elseif os(Linux)
+    return 1 //  wcwidth_l and Locale.ID_UTF8 not exposed from Glibc
```

Comment:
I think that we need to ensure that this is correct. This would otherwise mishandle emoji and CJK characters.